data/README.md

```diff
@@ -0,0 +1,163 @@
+
+_Kaveh Karimi & Lasse Keetz, 31-10-2021._
+
+# Instructions to create input data for a custom site
+
+The scripts to create new input data with this `ctsm-api` version require the following global datasets and file structures:
+
+- `<cesm-data-root>/atm/datm7/atm_forcing.datm7.GSWP3.0.5d.v1.c170516`
+- `<cesm-data-root>/share/domains/domain.lnd.fv0.9x1.25_gx1v7.151020.nc`
+- `<cesm-data-root>/lnd/clm2/surfdata_map/release-clm5.0.18/surfdata_0.9x1.25_hist_16pfts_Irrig_CMIP6_simyr2000_c190214.nc`
+- `<cesm-data-root>/lnd/clm2/surfdata_map/release-clm5.0.18/surfdata_0.9x1.25_hist_78pfts_CMIP6_simyr2000_c190214.nc`
+
+The datasets are available at https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/.
+
+If you want to use different versions of these data (WARNING! Might break the model, only if you know what you are doing!), make sure they exist in the given paths and adapt the names of the files in:
+
+- `<ctsm-api-root>/data/create_data.py` -> Adapt name of atmospheric forcing
+- `<ctsm-root>/tools/site_and_regional/default_data.cfg` -> Adapt other file names
+
+This document provides a working example tested on a remote server ([SAGA](https://documentation.sigma2.no/hpc_machines/saga.html])) with Anaconda. The time it takes to create the data is dependent on the machine and the amount of cores you use; on Saga, creating data for a single site requires approximately 1.5 hours.
+
+## 1 Clone `CTSM` and `ctsm-api`, checkout externals
+
+Optional but recommended: also checkout the CTSM tag we have tested the scripts for.
+
+```
+git clone https://github.com/ESCOMP/CTSM.git  # CTSM
+### Optional start ###
+cd CTSM
+git checkout tags/ctsm5.1.dev112 -b subset-data
+cd ..
+### Optional end ###
+./CTSM/manage_externals/checkout_externals
+git clone https://github.com/NorESMhub/ctsm-api.git  # ctsm-api
+```
+
+## 2 Install/load Python dependencies
+
+You need a recent Python environment that has `xarray` with `netcdf4` installed to run the scripts.
+On SAGA:
+
+```
+# Load Anaconda module
+module load Anaconda3/2022.05
+# Install new conda environment
+conda create -n subset-data-env -c conda-forge python=3.10 xarray netCDF4
+# Activate conda environment. Try the commented out version if this throws an error.
+source activate subset-data-env  # conda activate subset-data-env
+```
+
+## 3 Add new site information to config file
+
+Edit `<ctsm-api-root>/data/sites.json` or use it as a template to create a new file.
+You need to provide `lon` and `lat` coordinates and a site name (convention: three capital letters plus optional integer). You can include one or multiple sites (see example below).

```

Comment:
I realized I haven't tested sites with special characters or space in their names. It might cause an issue!
Just to be safe, let's mention names must be alphanumeric with no spaces.